app/repositories/ApprovalRepository.scala

```diff
@@ -110,4 +111,29 @@ class ApprovalRepositoryImpl @Inject() (implicit mongoComponent: ReactiveMongoCo
     //$COVERAGE-ON$
   }
 
+  def changeStatus(id: String, statusInfo: ApprovalProcessStatusChange): Future[RequestOutcome[Boolean]] = {
```

Comment:
Think the Boolean should be Unit, it never returns false, either succeeds or fails